### ISSUE TITLE
PolymerBase.splice() TS declaration does not match the one from Polymer js

### ISSUE THREAD

**Issue by ndossougbete** [nippur72/PolymerTS#66]
The [Polymer documentation](https://www.polymer-project.org/1.0/docs/devguide/properties.html#array-mutation) says that splice can also be used to insert elements. Here is the signature it gives:

    splice(path, index, removeCount, [item1, ..., itemN])

The declaration in `polymer-ts.ts` is missing the items to add:

    splice(path: string, start: number, deleteCount: number):any;

**Comment by nippur72** [comment]
I think there's an glitch in the documentation as the [API](https://polymer.github.io/polymer) declares it differently, and also the source code:  

```js
splice: function (path, start, deleteCount) {
   var array = this.get(path);
   var args = Array.prototype.slice.call(arguments, 1);
   var ret = array.splice.apply(array, args);
   this._notifySplice(array, path, start, args.length - 2, ret);
   return ret;
}
```

**Comment by ndossougbete** [comment]
It uses `arguments` though. When I called it with extra items it worked as explained in the documentation

**Comment by nippur72** [comment]
I hate when declaration are so ambiguous :-)

I've fixed it in 0.1.22.

**Comment by ndossougbete** [comment]
Which is why es2015's `...args` is going to help a lot. Thanks for the fix!
